components/brave_wallet/browser/internal/hd_key_sr25519.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_SR25519_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_SR25519_H_
+
+#include <array>
+#include <memory>
+
+#include "base/containers/span.h"
+
+namespace brave_wallet {
+
+namespace schnorrkel {
```

Comment:
I believe it would be better to use some more general namespace, like polkadot

Reply:
I had a really long conversation with Brian Johnson and he suggested just doing away with the namespace altogether so this is the ultimate direction that I took.